cpp/include/cuspatial/cuspatial.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+namespace cuspatial
+{
+	struct location_3d
+	{
+		double latitude;
+		double longitude;
+		double altitude;
+	};
+
+	struct coord_2d
+	{
+		double x;
+		double y;
+	};
+
+	struct TimeStamp
```

Comment:
its is an unfortunate acronym. Makes it look like "It's a timestamp".  What's wrong with `timestamp`? It's in the cuspatial namespace so it can't be confused with a `cudf::timestamp`, they are different types and namespaces.